src/pages/index.js

```diff
@@ -355,134 +378,153 @@ const Index = ({ data: initialData }) => {
   const onAdError = (e) => {
     logger.error(e)
   }
+
+  const onCompletedOnboarding = async () => {
+    await SetHasViewedIntroFlowMutation({ enabled: true, userId: userGlobalId })
+    setHasViewedIntroFlow(true)
+  }
   return (
     <div className={classes.pageContainer} data-test-id="new-tab-page">
-      {enableBackgroundImages ? (
-        <UserBackgroundImageContainer user={user} />
-      ) : null}
-      <div className={classes.fullContainer}>
-        <div className={classes.topContainer}>
-          <div className={classes.userMenuContainer}>
-            <UserImpactContainer
-              userId={userGlobalId}
-              userImpact={userImpact}
-              user={user}
-            />
-            <div>
-              <Typography variant="h5" className={clsx(classes.userMenuItem)}>
-                <MoneyRaisedContainer app={app} />
-              </Typography>
+      {!hasViewedIntroFlow ? (
+        <div className={classes.OnboardingFlow}>
+          <OnboardingFlow onComplete={onCompletedOnboarding} />
```

Comment:
Please add the logo on the top left, positioned the same as, e.g., the logo on the error page